packages/ignite-basic-structure/index.js

```diff
@@ -54,6 +54,9 @@ const add = async function (context) {
     filesystem.copy(`${sourceFolder}/App`, `${process.cwd()}/App`, { overwrite: true })
   }
 
+  // Create ./ignite/plugins
+  filesystem.dir(`${process.cwd()}/ignite/plugins`)
```

Comment:
1 line of code in the CLI gets us custom plugins.   Boosh!  I'm available to chat anytime.